buildmatrix/cli.py

```diff
@@ -217,7 +217,11 @@ def decide_what_to_build(recipes_path, python, packages, numpy):
     # logger.info('{: <8} | {}'.format('to build', 'built package name'))
     recipes_path = os.path.abspath(recipes_path)
     logger.info("recipes_path = {}".format(recipes_path))
-    for folder in sorted(os.listdir(recipes_path)):
+    if 'meta.yaml' in os.listdir(recipes_path):
+        folders = [recipes_path]
+    else:
+        folders = sorted(os.listdir(recipes_path))
```

Comment:
Ah I see that it is *just* out of sight because of the code folding.  Lines 228-229 are this:

```python
        if 'meta.yaml' not in os.listdir(recipe_dir):
            continue
```